platforms/ios/Tangram.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>NextzenApiKey</key>
```

Comment:
Should we also include `NextzenApiKey` variable here in the plist file?

Reply:
Ahh, this is just github being weird. Its actually a plist for the workspace and not the demo!

Reply:
Wow I don't know how git saw this as a move.. just consider this file as deleted, the new one is totally unrelated.